File: app/src/main/java/com/dhsdevelopments/potato/imagecache/LoadQueueEntry.java
package com.dhsdevelopments.potato.imagecache;

class LoadQueueEntry
{
    String url;
    int imageWidth;
    int imageHeight;
    StorageType storageType;
    BitmapCacheEntry bitmapCacheEntry;
    String apiKey;

    LoadQueueEntry( String url, int imageWidth, int imageHeight, StorageType storageType, BitmapCacheEntry bitmapCacheEntry, String apiKey ) {
        this.url = url;
        this.imageWidth = imageWidth;
        this.imageHeight = imageHeight;
        this.storageType = storageType;
        this.bitmapCacheEntry = bitmapCacheEntry;
        this.apiKey = apiKey;
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/imagecache/LoadQueueEntry.kt
package com.dhsdevelopments.potato.imagecache

internal class LoadQueueEntry(
        var url: String,
        var imageWidth: Int,
        var imageHeight: Int,
        var storageType: StorageType,
        var bitmapCacheEntry: BitmapCacheEntry,
        var apiKey: String?)
